backend/pkg/plugins/plugins.go

```diff
@@ -77,20 +77,30 @@ func periodicallyWatchSubfolders(watcher *fsnotify.Watcher, path string, interva
 	}
 }
 
-// GeneratePluginPaths takes the staticPluginDir and pluginDir and returns a list of plugin paths.
-func GeneratePluginPaths(staticPluginDir string, pluginDir string) ([]string, error) {
+// GenerateSeparatePluginPaths takes the staticPluginDir and pluginDir and returns a list of plugin paths.
```

Comment:
This says different to what it returns.

Reply:
Made this more descriptive to reflect the two lists